src/js/setup.js

```diff
@@ -31,6 +31,7 @@ const autoSetup = function() {
   // through each list of elements to build up a new, combined list of elements.
   const vids = document.getElementsByTagName('video');
   const audios = document.getElementsByTagName('audio');
+  const divs = document.getElementsByTagName('div');
```

Comment:
this allows for using `data-setup` with divs. Though, this seems a bit generic, should we use a different trigger with divs?